be/src/exec/workgroup/work_group_fwd.h

```diff
@@ -12,4 +12,10 @@ class ScanExecutor;
 
 using WorkGroupPtr = std::shared_ptr<WorkGroup>;
 
+// Two types of scan executor: OlapScan and HdfsScan
+enum ScanExecutorType {
+    TypeOlapScanExecutor,
+    TypeHdfsScanExecutor,
```

Comment:
What about other scan executor?

Reply:
Current there're only two kinds of ScanExecutor.